python/cudf/bindings/cudf_cpp.pyx

```diff
@@ -156,7 +156,7 @@ cdef gdf_column* column_view_from_column(col):
         else:
             data_ptr = 0
 
-    if col._mask is not None and col.null_count > 0:
+    if col._mask is not None and col.null_count >= 0:
```

Comment:
Can you explain this change? Will col.null_count ever be negative?  I think the reason for the previous logic was that if null_count was zero you didn't need a null mask.  Now it looks like it will always have a mask (which may be desirable, I just want to understand)?

Reply:
In ideal scenario, `col.null_count` will never be negative. 
If the input is like below:
```python
import cudf
sr = cudf.Series(list(range(5)))
print(sr.replace([3,4], None))
``` 
Then, mask for `sr` is not required but I am allocating the mask in python side before passing this to `gdf_find_and_replace_all()` because mask will be needed to clear bit corresponding to data element 3 and 4 to represent NULL.
That's why, equality is needed in `if col._mask is not None and col.null_count >= 0:`.